### PR TITLE
Print local backend command logs

### PR DESCRIPTION
**Pull request by 6543**
the cmd currently shows the full prompt and drop the exact error level.

this set the prompt to be hidden and let cmd exit with error level reported by the command

---
*Sponsored by Kithara Software GmbH*

### CODE REVIEW

**Comment by 6543** [comment]
draft untill tested ...

**Comment by 6543** [comment]
... tested multible times

File: pipeline/backend/local/command.go [hunk 1]
```diff
@@ -31,10 +31,13 @@ func genCmdByShell(shell string, cmds []string) (args []string, err error) {
...
 	case "cmd":
-		script := ""
+		script := "@SET PROMPT=$\n"
 		for _, cmd := range cmds {
-			script += fmt.Sprintf("%s || exit 1\n", cmd)
+			script += fmt.Sprintf("@echo + %s\n", strings.TrimSpace(shellescape.Quote(cmd)))
+			script += fmt.Sprintf("@%s\n", cmd)
+			script += "@IF NOT %ERRORLEVEL% == 0 exit %ERRORLEVEL%\n"
 		}
+		// TODO: set temp dir
```
Comment (6543) [inline pipeline/backend/local/command.go:40]:
```suggestion
```

let's remove it for now ... an issue I'll work later on

**Comment by 6543** [comment]
# before
image
# now
image

**Review by anbraten — APPROVED** [review]
No idea from the cmd magic, but the screenshot looks promising.

**Pull request merged**